DS13/code/modules/overmap/combat.dm

```diff
@@ -0,0 +1,507 @@
+/mob/living/carbon/canMobMousedown(atom/object, location, params)
+	. = ..()
+	if(!overmap_ship)
+		return
+	. = overmap_ship
+
+
+/obj/structure/overmap
+	var/weapons_ready = TRUE //Are we ready to fire? This is a straight cooldown
+	var/weapons_cooldown = 25 //This prevents spam. Allows a shot once every 2 seconds
+	var/datum/beam/current_beam = null //Our phaser beam, wow!
+	var/damage = 0 //How much damage do we do? This is +10'd assuming weapons are powered!
+	var/charging = FALSE //charging weapons? if so, we can't fire
+	var/admin_override = FALSE //Need to test? set this on and you can fire and move yourself
+	var/fire_mode = "phaser"
+	var/photons = 4 //How many torpedoes do we have?
+	var/list/weapon_sounds = list('DS13/sound/effects/weapons/phaser.ogg','DS13/sound/effects/weapons/phaser2.ogg','DS13/sound/effects/weapons/phaser3.ogg','DS13/sound/effects/weapons/phaser4.ogg')
```

Comment:
this should probably be a global list

Reply:
If phasers use this list make it a global list.